### ISSUE TITLE
Windows MAX_PATH

### ISSUE THREAD

**Issue by mtveerman** [jmirtsch/GitIFCPrototype#1]
Because entities are stored hierarchically, you might hit the windows MAX_PATH length sooner or later....

**Comment by jmirtsch** [comment]
Absolutely, this is a real issue.  Windows10 supports long file paths but this needs consideration.

**Comment by mtveerman** [comment]
You need to opt-in for this on Win10. It's not supported by default.
https://msdn.microsoft.com/en-us/library/aa365247(VS.85).aspx#maxpath
